twake/frontend/src/app/scenes/Apps/Drive/Viewer/Viewer.js

```diff
@@ -103,7 +103,7 @@ export default class Viewer extends Component {
           <div className="view_header">
             <div className="title">{current.name}</div>
 
-            {editor_candidate.length == 1 && (
+            {DriveService.previewonly === false && editor_candidate.length == 1 && (
```

Comment:
`!DriveService.previewonly` should be better than comparing to `false`

Reply:
Done